### PR TITLE
Make forms error message adhere to styleguide

### PR DESCRIPTION
**Pull request by huangy22**
For Refs #3167

Edit the form error messages to adhere to the styleguide:
- Removes instances of "Please"
- Removes . on short single sentences

### CODE REVIEW

File: tests/unit/test_forms.py [hunk 1]
```diff
@@ -69,12 +69,12 @@ def test_good_passwords(self, password):
...
             (
                 "qwerty",
-                ("This is a top-10 common password. Add another word or two. "
+                ("This is a top-10 common password. Add another word or two."
                  "Uncommon words are better."),
             ),
             (
                 "bombo!b",
-                ("Password is too easily guessed. Add another word or two. "
+                ("Password is too easily guessed. Add another word or two."
```
Comment (pradyunsg) [inline tests/unit/test_forms.py]: This doesn't seem right, there should be a space between the two sentences?

Comment (huangy22) [reply tests/unit/test_forms.py]: You are right. I made a mistake here. Sorry about that.

File: warehouse/accounts/forms.py [hunk 2]
```diff
@@ -145,13 +145,13 @@ def validate_email(self, field):
         if self.user_service.find_userid_by_email(field.data) is not None:
             raise wtforms.validators.ValidationError(
                 "This email address is already being used by another account. "
-                "Please use a different email."
+                "Use a different email."
             )
         domain = field.data.split('@')[-1]
         if domain in disposable_email_domains.blacklist:
             raise wtforms.validators.ValidationError(
                 "Sorry, you cannot create an account with an email address "
```
Comment (di) [inline warehouse/accounts/forms.py]: I think we could probably lose the "Sorry" here as well.

File: warehouse/accounts/forms.py [hunk 3]
```diff
@@ -84,13 +84,13 @@ def validate_password(self, field):
...
-                        "The password you have provided is invalid. Please "
-                        "try again."
+                        "The password you have provided is invalid. "
+                        "Try again."
                     )
             except TooManyFailedLogins:
                 raise wtforms.validators.ValidationError(
                     "There have been too many unsuccessful login attempts, "
-                    "please try again later."
+                    "Try again later."
```
Comment (di) [inline warehouse/accounts/forms.py]: We either need to put a period at the end of the previous phrase, or lowercase this `Try`.

**Comment by huangy22** [comment]
@di Thank you so much for the comments! I have been a little busy attending a data science conference recently, sorry for my reply. I have addressed your comments, let me know if there's anything else to change!

File: tests/unit/accounts/test_forms.py [hunk 4]
```diff
@@ -237,8 +237,8 @@ def test_blacklisted_email_error(self):
         assert not form.validate()
         assert (
             form.email.errors.pop() ==
-            "Sorry, you cannot create an account with an email address from "
-            "this domain. Please use a different email."
+            "Sorry, you can't create an account with an email address from "
+            "this domain. Use a different email."
```
Comment (di) [inline tests/unit/accounts/test_forms.py]: This should be:
```diff
-            "Sorry, you cannot create an account with an email address from "
-            "this domain. Please use a different email."
+            "You cannot create an account with an email address from "
+            "this domain. Use a different email."
```

**Comment by di** [comment]
@huangy22 I'm still seeing a lot of changes in the following files that are unnecessary:

* All the changes in `tests/unit/test_forms.py`
* All the changes in `tests/unit/test_views.py`

**Comment by huangy22** [comment]
@di Thank you so much for your comments and accepting the PR! I have one question though, somehow I seem to not be able to find myself on the contributor page and I wonder if there's any steps I'm missing here?

**Comment by di** [comment]
@huangy22 If you're talking about https://github.com/pypa/warehouse/graphs/contributors, it seems like that page only shows at most 100 contributors ordered by number of commits (even though we have 104).

Since each PR gets squashed, this one only counts as a single commit. My guess is since your contribution was relatively recent, you're one of the 4 contributors that don't show up on that page.

**Pull request merged**